src/sniffer.c

```diff
@@ -626,8 +638,12 @@ void ssl_InitSniffer_ex(int devId)
 {
     wolfSSL_Init();
     wc_InitMutex(&ServerListMutex);
+#ifndef HAVE_C___ATOMIC
```

Comment:
Maybe you could put the #ifdef-ed locking and unlocking of SessionMutex under a LOCK_SESSION / UNLOCK_SESSION pair of macros?

Reply:
Done